packages/webpack-plugin/test/e2e/projects/emit-info-diagnostic/webpack.config.js

```diff
@@ -9,17 +9,6 @@ module.exports = {
     plugins: [
         new StylableWebpackPlugin({
             diagnosticsMode: 'strict',
```

Comment:
this is here to show that it doesn't break in strict mode with info diagnostics?

Reply:
yep